src/lib/template/dom-repeat.html

```diff
@@ -253,6 +316,50 @@
       }
     },
 
+    _limitChanged: function(limit) {
+      if (this.items) {
+        this._debounceTemplate(this._render);
+      }
+    },
+
+    _computeFrameTime: function(rate) {
+      return Math.ceil(1000/rate);
+    },
+
+    _initializeChunkCount: function() {
+      if (this.initialCount) {
+        this.limit = this.initialCount;
+        this._chunkCount = parseInt(this.chunkCount, 10) || this.initialCount;
```

Comment:
Don't init chunk count here (unnecessary); measure initial render time & use for first chunk